modules/core/client/config/core.client.routes.js

```diff
@@ -15,28 +19,52 @@ angular.module('core').config(['$stateProvider', '$urlRouterProvider',
     $stateProvider
     .state('home', {
       url: '/',
-      templateUrl: 'modules/core/client/views/home.client.view.html'
+      views: {
+        'hero': {
+          templateUrl: 'modules/core/client/views/hero.client.view.html'
+        },
+        'main': {
+          controller: 'HomeController',
```

Comment:
Yeah that makes sense and I will make that change for anything that currently has controllerAs.  I don't want to do it for anything that does not currently have controllerAs otherwise I will need to stylize the entire controller in this PR as well.  Right?